### DISCUSSION TITLE
大佬有没有兴趣做一下mactype的适配

### DISCUSSION THREAD

**Discussion by lurne** [F9y4ng/GreasyFork-Scripts#120] [问答专区(Question / Answer)]
自从用了这个的苹方字体+渲染，浏览器里赏心悦目。但是系统那边还是一塌糊涂，大佬有空能做一个Mactype的配置文件来么

**Comment by F9y4ng** [comment]
这个可真没有😅 你可以[搜索](https://www.google.com/search?hl=zh-CN&source=hp&safe=off&newwindow=1&q=Mactype%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6)一下其他人分享的配置文件。

## 但是
此类配置文件应该与 **字体渲染脚本的设置备份文件** 都有一样的问题：即每个配置文件都与其使用者本地的显示器、显卡、计算机相关的参数设置有关，分享配置文件并不意味着在别人计算机上的效果好就一定会在你的计算机上也会好。

所以，建议你还是花点心思慢慢调试适合自己计算机的配置参数。

**Reply by lurne** [reply]
多谢大佬的辛勤劳动，网页渲染用了三天，舒服斯基。然后才想起用win7的时候玩过那个软件。这么多年忘掉了，现在27寸的4k开100%还是有点细
那我还是照着网页里的效果一点点调吧
